testsuite/scripts/tutorials/test_05-raspberry_electrophoresis.py

```diff
@@ -17,15 +17,26 @@
 
 import unittest as ut
 import importlib_wrapper
+import numpy as np
 
 tutorial, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
     "@TUTORIALS_DIR@/05-raspberry_electrophoresis/05-raspberry_electrophoresis.py",
-    gpu=True, num_iterations=10, num_steps_per_iteration=10)
+    gpu=True, box_l=20., E=0.2, num_iterations=150, num_steps_per_iteration=400)
```

Comment:
With an E-field < 0.2, the raspberry frequently travels a shorter distance along the axis field. Maybe the thermostat is too strong. Even with an E-field of 0.2, the random movement on the 2 other axes can still be larger than the E-field axis, requiring an annoying `min` function:
https://github.com/espressomd/espresso/blob/211fe9962130b8dadd3fb78e5aa5b983933d448e/testsuite/scripts/tutorials/test_05-raspberry_electrophoresis.py#L43
instead of the more correct solution
https://github.com/espressomd/espresso/blob/211fe9962130b8dadd3fb78e5aa5b983933d448e/testsuite/scripts/tutorials/test_05-raspberry_electrophoresis.py#L35-L36
I ran this test in a loop with different `E`, `num_iterations`, and `num_steps_per_iteration` to gather statistics and this inelegant solution is the one that didn't fail at random.